packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -242,215 +235,261 @@ type JobAdData = {
   salary?: string,
   location: string,
   roleType?: string,                 // i.e. part-time, contract
-  deadline?: moment.Moment,          // not displayed, only used to hide the ad after this date
+  deadline?: moment.Moment,          // also used to hide the ad after this date
   getDescription: (classes: ClassesType) => JSX.Element
 }
 
 // job-specific data for the ad
-// (also used in the confirmation email, so links in the description need to be absolute)
+// (also used in the reminder email, so links in the description need to be absolute)
 export const JOB_AD_DATA: Record<string, JobAdData> = {
```

Comment:
If we decide to keep doing job ads, we'll probably want to move this data to its own collection. For now it seemed premature, and this PR is large enough already.